### PR TITLE
Add tutorial resource

### LINKED ISSUE #1399: Create a tutorial_steps resource
Projects are getting simple Chimp&See-style popup tutorials.

We'll need a tutorial_steps resource, each with its own `media` and a `content`.

Media will live in the project's attached media, so maybe that should be a link?

Content'll just be Markdown.

They should be reorderable. Do linked resources maintain their order when it changes? Otherwise we could have an explicit `index` field or something.

### PR DESCRIPTION
**Pull request by edpaget**
Closes #1399

### CODE REVIEW

File: app/models/project.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Project < ActiveRecord::Base
 
   EXPERT_ROLES = [:expert, :owner]
 
+  belongs_to :default_tutorial, class_name: "Tutorial"
```
Comment (camallen) [inline app/models/project.rb]: should this association be the other way around? `Project has_one :default_tutorial` and a `Tutorial belongs_to :project`?

Comment (edpaget) [inline app/models/project.rb]: Why? The key is on the project table.

Comment (camallen) [inline app/models/project.rb]: i thought it would clean up the inverse relation FK specification.

Comment (edpaget) [inline app/models/project.rb]: Sure if you'd prefer it the other way I'll switch it around. It made more sense to me that the project had the default defined on it, but I can see the other way too.

Comment (camallen) [inline app/models/project.rb]: i really don't mind. just asked the question. you decide

Comment (edpaget) [inline app/models/project.rb]: Up to you man. You're going to be maintaining it. I thought I answered the question in my post above, that I think it makes sense that the project holds the reference to its default tutorial, but I could see it either way.

Comment (camallen) [inline app/models/project.rb]: leave it

**Comment by camallen** [comment]
This LGTM, you happy @edpaget?

**Comment by edpaget** [comment]
Yup

**Pull request merged**